### PR TITLE
setAsReady mutation

### PR DESCRIPTION
**Pull request by akaoss**
```graphql
mutation markAsReady {
  markAsReady(input: { boardId: 2, ready: true }) {
    ready
  }
}
```

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -4,49 +4,49 @@
   end
 
   post "/graphql", to: "graphql#execute"
-  root 'boards#new'
+  # root 'boards#new'
```
Comment (tuykin) [inline config/routes.rb:7]: Maybe we can do this change in separate PR? 🙈 
And why not totally remove it?

Comment (akaoss) [reply config/routes.rb:7]: We'll refactor it later.
